src/mysql_router_helpers.py

```diff
@@ -148,7 +149,11 @@ def bootstrap_and_start_mysql_router(
             f"{port}",
             "--conf-set-option",
             "DEFAULT.server_ssl_mode=PREFERRED",
-        )
+            "--conf-use-gr-notifications",
+        ]
+
+        if force:
+            bootstrap_mysqlrouter_command.append("--force")
```

Comment:
Pass the force flag depending so that only reconfigurations overwrite the existing config.